src/nvim/api/win_config.c

```diff
@@ -212,6 +212,10 @@
 ///   - hide: If true the floating window will be hidden.
 ///   - vertical: Split vertically |:vertical|.
 ///   - split: Split direction: "left", "right", "above", "below".
+///   - cmdline: If true, indicate that this window is used for ext_cmdline.
+///     Nvim will anchor the 'wildoptions' popupmenu to this window.
```

Comment:
Not clear from this description what the window shows. Is it only for the popup menu? `ext_cmdline` presents the entire cmdline, not only the popup menu.

Reply:
What the window contains is entirely up to the cmdline implementation. This just allows the implementation to "mark" the window that is used to display the cmdline as such, to allow Nvim to do some work with it internally.

For now that just means anchoring the completion popupmenu, not sure if anything else could be done. Originally also thought this could be used to redraw and move the cursor to it, but since we can do that with `nvim__redraw` this is no longer necessary.